themes/digital.gov/layouts/communities/single.html

```diff
@@ -131,6 +131,12 @@ <h3>Community Managers</h3>
             <div class="content">
               {{- .Content -}}
             </div>
+            {{- if in .Params.topics "community-ux" -}}
```

Comment:
I don't think the topic tag should be hardcoded in this template. How could it be made more generic?
We could change the community topic tag to more closely match the community slug. Would that work?